apps/mobile/mobile-app/components/project-details/tab-content/overview.tsx

```
@@ -0,0 +1,43 @@
+
+			<Text className="text-lg font-bold mb-2">Our Mission</Text>
+			<Text className="text-gray-600">
+				We're dedicated to providing innovative solutions that address key
+				challenges in our field. Our team brings together expertise and passion
+				to ensure successful outcomes.
+			</Text>
+			<Text className="text-lg font-bold mb-2 mt-3">
+				How Funds Will Be Used
+			</Text>
+			{items.map((item, index) => (
+				<View key={index} style={{ flexDirection: 'row', marginBottom: 8 }}>
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)